docs/_docs/themes.md

```diff
@@ -4,28 +4,37 @@ title: Themes
 permalink: /docs/themes/
 ---
 
-Jekyll has an extensive theme system, which allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package layouts, includes, and stylesheets in a way that can be overridden by your site's content.
+Jekyll has an extensive theme system that allows you to leverage community-maintained templates and styles to customize your site's presentation. Jekyll themes package layouts, includes, and stylesheets in a way that can be overridden by your site's content.
 
-## Installing a theme
+## Understanding gem-based themes
 
-1. To install a theme, first, add the theme to your site's `Gemfile`:
+When you [create a new Jekyll site](/docs/quickstart) (by running the `jekyll new <PATH>` command), Jekyll installs a gem-based theme called [Minima](https://github.com/jekyll/minima). 
 
-        gem 'my-awesome-jekyll-theme'
+With [gem-based themes](/docs/themes/), some of the theme directories and files are stored in the gem, hidden from view in your Jekyll project. As a result, the files and directories shown for your site are a subset of all the theme's files. In the case of Minima, you see only the following:
 
-2. Save the changes to your `Gemfile`
-3. Run the command `bundle install` to install the theme
-4. Finally, activate the theme by adding the following to your site's `_config.yml`:
+```
+├── Gemfile
+├── Gemfile.lock
+├── _config.yml
+├── _posts
+│   └── 2016-12-04-welcome-to-jekyll.markdown
+├── about.md
+└── index.md
+```
 
-        theme: my-awesome-jekyll-theme
+The Gemfile and Gemfile.lock files are used by Bundler to keep track of the required gems and gem versions you need to build your Jekyll site.
 
-You can have multiple themes listed in your site's Gemfile, but only one theme can be selected in your site's `_config.yml`.
-{: .note .info }
+Gem-based themes make it easy for theme developers to remotely update the theme for everyone who has the gem. When there's an update, theme developers push the update to RubyGems. 
+
+You can then run `bundle update` to update all gems in your project (or run `bundle update <theme>`, replacing `<theme>` with the theme name, such as `minima`, to just update your theme gem). Any new files or updates the theme developer has made (such as to stylesheets or includes) will be pulled into your project automatically.
 
-## Overriding theme defaults
+The goal of gem-based themes is to allow you to get all the benefits of a robust, continually updated theme without having all the theme's files getting in your way and over-complicating what might be your primary focus: creating content.
 
-Jekyll themes set default layouts, includes, and stylesheets, that can be overridden by your site's content. For example, if your selected theme has a `page` layout, you can override the theme's layout by creating your own `page` layout in the `_layouts` folder (e.g., `_layouts/page.html`).
+## Overwriting theme defaults
```

Comment:
I think **Installing a theme** needs to come before **Overriding theme defaults**.

Before the ambitious-but-naïve user starts hacking away, it's probably a good idea to see if a theme already exists that serves their needs.